CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Fixed
+- deprecated `store::UnorderedMap` due to its fragmentation property ([#1139](https://github.com/near/near-sdk-rs/pull/1139))
+- deprecated `store::UnorderedSet` due to its fragmentation property ([#1139](https://github.com/near/near-sdk-rs/pull/1139))
```

Comment:
We don't need to manually edit CHANGELOG anymore, just write it in the PR description and release-plz will include it in the CHANGELOG, see the automated PR: https://github.com/near/near-sdk-rs/pull/1138